build.gradle

```diff
@@ -62,6 +62,7 @@ def opensearch_tmp_dir = rootProject.file('build/private/opensearch_tmp').absolu
 opensearch_tmp_dir.mkdirs()
 
 ext {
+    System.setProperty("OPENSEARCH_INITIAL_ADMIN_PASSWORD", "myStrongPassword123!")
```

Comment:
This would not work for versions <2.12 because at line 89, 91, 306 where its actually using the password, the expected password is `admin` for <2.12 but it will use `myStrongPassword123!` instead

Reply:
So shall I put the version check?

Reply:
yes..that would resolve any issues with password for <2.12